cleanup.yml

```diff
@@ -86,3 +86,85 @@
       vars:
         name_prefix: "{{ ibmcloud_vpc_iso_name_prefix }}"
         ibmcloud_ic_region: "{{ ibmcloud_vpc_iso_region }}"
+
+- name: Cleanup IBM Cloud VPC VSI Related to Isolated Nodes
```

Comment:
I believe it is duplicated here

Reply:
It is and I removed the duplication in the latest commit but for some reason its not picking up in PR. I dont know why.